pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -519,20 +522,15 @@ def disconnect(self):
 
         Must be connected to the server when calling this method.
         """
-        # print("Debug: PywbemServer disconnect")
-
-        # TODO(PR900): Remove the PR900 flag test and else path once done.
-        if PR900_REQUIRE_CORRECT_CONNECTION_STATUS:
-            assert self._wbem_server is not None  # Must be connected
-        else:
-            # Old code to be removed once done
-            if self._wbem_server is None:
-                return
+        ctx = click.get_current_context()
+        if ctx.obj:
+            if ctx.obj.verbose:
+                click.echo("Disconnect from server {}".
+                           format(self._mock_server or self._server))
```

Comment:
The term "server" is ambiguous here. I suggest to be specific about mock environment vs. WBEM server, e.g. by changing the lines with the click.echo() call to:
```
            if self._mock_server:
                server_txt = "mock environment {}".format(self._mock_server)
            else:
                server_txt = "WBEM server {}".format(self._server)
            click.echo("Disconnecting from {}".format(server_txt))
```

Reply:
DONE